### PR TITLE
Skip the first prompt of `pdm init`

### PR DESCRIPTION
**Pull request by ulwlu**
## Pull Request Check List

- [x] A news fragment is added in `news/` describing what is new.
- [x] Test cases added for changed code.

## Describe what you have changed in this PR.

- Achieve this proposal https://github.com/pdm-project/pdm/discussions/350
- skip the first prompt of `pdm init`

### CODE REVIEW

**Comment by ulwlu** [comment]
Tests are failed on other PR as well, but seems recovered minutes ago.

**Comment by frostming** [comment]
Yes, blocked by #351 , will merge after that, thanks for the contribution,

File: pdm/cli/actions.py [hunk 1]
```diff
@@ -426,9 +429,7 @@ def version_matcher(py_version):
         dict.fromkeys(filter(version_matcher, project.find_interpreters(python)))
     )
     if not found_interpreters:
-        raise NoPythonVersion(
-            f"Python interpreter {python} is not found on the system."
-        )
+        raise NoPythonVersion("Python interpreter is not found on the system.")
     if first or len(found_interpreters) == 1:
         selected_python = found_interpreters[0]
     else:
```
Comment (frostming) [inline pdm/cli/actions.py:435]: I would like to move line 421 to after this line so that it only displays when a decision is actually needed.

Comment (ulwlu) [reply pdm/cli/actions.py:434]: fixed it here.
2bd80fb

File: news/352.feature.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Skip the first prompt of `pdm init`
```
Comment (frostming) [inline news/352.feature.rst:1]: Sorry for the misleading filename, it should be named `352.feature.md`.

**Comment by frostming** [comment]
Would you please merge or rebase the master branch, the CI failure should be fixed.

**Comment by frostming** [comment]
Thank you for taking care of this, here you go.

**Comment by ulwlu** [comment]
Thank you for reviewing.

**Comment by 1041787154** [comment]
你好作者，十分欣赏您的作品，我在使用pdm管理python项目时，遇到个问题，请指教：
我的项目是多人合作项目，当我从Git上下载下最新的项目代码时，不能直接pdm sync（项目已有mock和toml后缀文件），报错为The package is not a valid PDM project.但是其他同事却能直接pdm sync。是什么问题呢？麻烦在有空时及时回复，谢谢

**Comment by frostming** [comment]
> 你好作者，十分欣赏您的作品，我在使用pdm管理python项目时，遇到个问题，请指教：
> 
> 我的项目是多人合作项目，当我从Git上下载下最新的项目代码时，不能直接pdm sync（项目已有mock和toml后缀文件），报错为The package is not a valid PDM project.但是其他同事却能直接pdm sync。是什么问题呢？麻烦在有空时及时回复，谢谢

你描述的事情全是代指 没有一个实例实锤 没法看 下次别这么提问题

**Pull request merged**